backend/app/tests.py

```diff
@@ -1,14 +1,49 @@
 """
-Tests for the main app.
+Tests for the gender analysis web app.
 """
 
 from django.test import TestCase
+from django.core.exceptions import ObjectDoesNotExist
+from .models import (
+    Pronoun,
+)
+
+
+class PronounTestCase(TestCase):
+    """
+    TestCase for the Pronoun model
+    """
+
+    def setUp(self):
+        Pronoun.objects.create(identifier='he', type='subj')
+        Pronoun.objects.create(identifier='him', type='obj')
+        Pronoun.objects.create(identifier='HIS', type='pos_det')
+        Pronoun.objects.create(identifier='his', type='pos_pro')
+        Pronoun.objects.create(identifier='himself', type='reflex')
+
+    def test_models_save(self):
+        he = Pronoun.objects.get(identifier='he')
+        self.assertEqual(str(he), 'Pronoun: he\nType: Subject')
+        self.assertEqual(type(he.type), str)
+        he.save()
```

Comment:
wonder why are you using .save() in here and above - is it possible to omit the method since I don't see a super strong need to touch the database? at least I didn't see he being used multiple times throughout, and perhaps it'll make the test more concise if we don't have the save() but please correct me if you have other ideas in mind!

Reply:
I agree that the `.save()` call here is unnecessary, unless I'm missing something obvious, since we assert that the custom field class is being utilized correctly during `.save()` below with the `his` `Pronoun` instance.